index.bs

```diff
@@ -2341,31 +2346,31 @@ To <dfn>obtain a set of possible trigger states</dfn> given a [=randomized respo
         with repetition.
 1. Return |possibleValues|.
 
-To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized response output configuration=] |config| and a double |epsilon|:
-
-1. Let |possibleValues| be the result of [=obtaining a set of possible trigger states=] with |config|.
-1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
-1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
+To <dfn>obtain a randomized source response pick rate</dfn> given a positive integer |states| and a double |epsilon|:
+1. Return |states| / (|states| - 1 + e<sup>|epsilon|</sup>).
 
-To <dfn>obtain a randomized source response</dfn> given a [=randomized response output configuration=] |config| and a double |epsilon|:
+To <dfn>obtain a randomized source response</dfn> given a positive integer |states| and a [=randomized response output configuration=] |config| and a double |epsilon|:
```

Comment:
Can we just pass the possible states instead? “Obtain a set of possible trigger starts” actually returns a set instead of an integer.